packages/psss/src/views/Tabs/AlertsTabView.js

```diff
@@ -19,41 +31,62 @@ const DateSubtitle = styled(Typography)`
 
 export const AlertsTabView = React.memo(() => {
   const [isPanelOpen, setIsPanelOpen] = useState(false);
-  const { countryCode } = useParams();
+  const [isPickerOpen, setIsPickerOpen] = useState(false);
+  const [rowData, setRowData] = useState({});
+  const [period, setPeriod] = useState(getCurrentPeriod());
+  const { organisationUnit: countryCode, syndromeName } = rowData;
```

Comment:
As far as I can tell, the `countryCode` wont exist in the rowData for single country users. I'm having trouble with my migrations so I can't test it. Does it work for you?

Reply:
Also I think it would be nice to get the period from the rowData if it's easy. I think it would be more robust and less likely to potentially cause bugs as the row clicked on is always the data used. ie. in the original designs it was possible to see all the alerts in the table so there could be multiple alerts from a single country.

Reply:
Both good points! I'm going to pass the period from `rowData` in now, and if you don't mind, I'll fix the `countryCode` issue in the PR for https://github.com/beyondessential/tupaia-backlog/issues/1511 - there is a bit of params refactoring that needs to happen for  that one

Reply:
Sweet - sounds good